doc/api/cli.md

```diff
@@ -194,7 +194,7 @@ changes:
     description: Paths delimited by comma (`,`) are no longer allowed.
 -->
 
-> Stability: 1.1 - Active development
+> Stability: 2.0 - Stable.
```

Comment:
You probably want to add a history entry in the YAML comment to track when that happened
```suggestion
> Stability: 2 - Stable
```